rose/client/game.py

```diff
@@ -60,18 +62,19 @@ def car(self):
     # Handling client events
 
     def client_connected(self):
-        print 'client connected: joining as', self.name
+        log.info('client connected: joining as %s', self.name)
         msg = message.Message('join', {"name": self.name})
         self.client.send_message(msg)
 
     def client_disconnected(self, reason):
-        print 'client disconnected:', reason.getErrorMessage()
+        log.info('client disconnected: %s', reason.getErrorMessage())
 
     def client_failed(self, reason):
-        print 'client failed:', reason.getErrorMessage()
+        log.info('client failed: %s', reason.getErrorMessage())
 
     def client_error(self, error):
-        print 'client error:', error.get('message')
+        log.info('client error: %sp', error.get('message'))
+        log.info('client error: %sp', error.get('message'))
```

Comment:
Two issues:
- log line duplicated
- trailing "p" at the end of the format string.